### DISCUSSION TITLE
Adding a "Request reinforcement" for friendlies

### DISCUSSION THREAD

**Discussion by XDSenDX** [Vdauphin/HeartsAndMinds#1549] [Question and answer] [answered]
Hello i want to add a reinforcement system which will spawn bots at base for friendlies. Is it possible?

Cheers.

**Comment by Vdauphin** [comment, marked as answer]
Hello,
Yes
Cheers

**Comment by XDSenDX** [comment]
I'll ask someone to help me with this.

Thank you.
